### DISCUSSION TITLE
Add option for storing disk size in MB or GB

### DISCUSSION THREAD

**Discussion by swe-mbernhard** [bb-Ricardo/netbox-sync#416] [Ideas]
From netbox v4.1.0 this changed:
The unit size for VirtualMachine.disk and VirtualDisk.size has been changed from 1 gigabyte to 1 megabyte. Existing values will be adjusted automatically during the upgrade process.

Is it possible to add this as an option in the settings file, so the script can handle either one unit?

**Comment by bb-Ricardo** [comment]
Hi, this issue has ben resloved in the development branch, check #400 and #413

**Reply by swe-mbernhard** [reply]
Thanks, and sorry for the ignorance!

**Reply by bb-Ricardo** [reply]
Don't be sorry. I should have left the issue open.

Let me know if the development branch works for you.

**Reply by swe-mbernhard** [reply]
The devel branch works just fine!
Thanks for the efforts you are making - very useful piece of software.
